src/lib/error-rate-check.js

```diff
@@ -20,11 +20,11 @@ module.exports = (appName, opts) => {
 	const threshold = opts.threshold || DEFAULT_THRESHOLD;
 	const samplePeriod = opts.samplePeriod || DEFAULT_SAMPLE_PERIOD;
 
-	let region = process.env.REGION ? '_' + process.env.REGION : '';
+	const region = process.env.REGION ? '_' + process.env.REGION : '';
 
 	return nHealth.runCheck({
-		id: 'error-rate',
-		name: `Error rate: greater than ${threshold}% of requests for ${appName}`,
+		id: `error-rate-${region}`,
```

Comment:
does line 23 mean that this will be `error-rate-_EU`?

Reply:
😮 I didn't realise that, thank you to catch it!